packages/verification/src/models/v1/flashcall-verification-status-response/flashcall-verification-status-response.ts

```diff
@@ -0,0 +1,31 @@
+import { CallResult, ReasonEnum, SourceEnum, VerificationStatusEnum } from '../enums';
+import { Identity } from '../identity';
+import { VerificationPriceCall } from '../verification-price-call';
+
+export interface FlashCallVerificationStatusResponse {
+
+  /** The unique ID of the verification request. */
+  id?: string;
+  /** The method of the verification request. This will always be `flashCall`. */
+  method?: 'flashcall';
```

Comment:
Same here:  doc say `flashCall`
if doc is wrong we should notify them

Reply:
Same behaviour has before: the server sends lowercase:
```json
{
  "id": "018ef61d-9c94-6f13-bd58-ebc87c2883b1",
  "method": "flashcall",
  "status": "FAIL",
  "reason": "Expired",
...
}
```